util/template.go

```diff
@@ -131,10 +132,21 @@ func toHCLAssignment(name string, value any) string {
 		return ""
 	}
 
-	rootBlockBody.SetAttributeValue(name, ctyVal.Value)
-	return string(f.Bytes())
+    rootBlockBody.SetAttributeValue(name, ctyVal.Value)
+
+    // Convert the file to a string
+    hclStr := string(f.Bytes())
+
+    // Conditionally wrap in "merge(var.tags, ...)" only if the attribute name is "tags"
+    if name == "tags" {
+        wrappedHcl := fmt.Sprintf("%s = merge(var.tags, %s)", name, hclStr[len(name)+3:])
+        return wrappedHcl
+    }
+    return hclStr
```

Comment:
that is something given by chatgpt .. it adds condition for tags, and add merge(var.tags ...) into it

Reply:
yeah, I'm going to think of a better way to handle this.. also, the indentation is wrong, make sure to have go fmt enabled?